codec-http2/src/main/java/io/netty/handler/codec/http2/HpackDecoder.java

```diff
@@ -553,4 +517,35 @@ static long decodeULE128(ByteBuf in, long result) throws Http2Exception {
         REQUEST_PSEUDO_HEADER,
         RESPONSE_PSEUDO_HEADER
     }
+
+    private interface Sink {
+        void appendToHeaderList(CharSequence name, CharSequence value);
+    }
+
+    private static class Http2HeadersSink implements Sink {
```

Comment:
final ?

Reply:
I don't tend to put `final` on private classes, but sure. Done.